README.md

```diff
@@ -6,5 +6,15 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+Rails is a framework for creatign new projects in Ruby
```

Comment:
This is true, but Rails isn't used for just any projects - it's specifically used for building back-end and full-stack web applications. Rails provides us with architecture and pre-existing files that we can inherit from, and this allows us to spin up new applications very quickly.